src/main/java/uk/gov/hmcts/divorce/print/BulkPrintService.java

```diff
@@ -48,4 +62,21 @@ public UUID print(Print print) {
                 )))
             .letterId;
     }
+
+    private byte[] getDocumentBytes(final String docUrl, final String authToken)  {
+        final var userAuth = request.getHeader(AUTHORIZATION);
+        final var userDetails = idamService.retrieveUser(userAuth).getUserDetails();
+        var resourceResponseEntity = documentManagementClient.downloadBinary(
```

Comment:
it is not clear what is the output type so we should not use var here

Reply:
Changed to specific type.